sortinghat/core/models.py

```diff
@@ -233,8 +233,12 @@ def __str__(self):
         return '%s - %s' % (self.individual.mk, self.organization.name)
 
 
-class MatchingBlacklist(EntityBase):
-    excluded = CharField(max_length=MAX_SIZE_CHAR_FIELD, primary_key=True)
+class RecommenderExclusionList(EntityBase):
```

Comment:
I think we should rename this to `RecommenderExclusionTerm` because the object stores the term. It's not a list.